lib/puppet/module_tool/skeleton/templates/generator/Gemfile

```diff
@@ -1,7 +1,14 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.3']
+gem 'metadata-json-lint'
```

Comment:
@hunner I'm definitely not a fan of the dependency chain that you suggest, though for a system whose purpose is to test Puppet modules, this is perfectly OK with me, as you only need it on a system who is checking the validity of a module. To that end, since we have a tool to check the metadata, we should use it. I know I've certainly screwed mine up before and the metadata-json-lint has become indispensable for me. It is also used for the puppetdb and xinetd modules.